coderd/database/migrations/000223_notifications_metrics.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE notification_messages
+ADD COLUMN queued_seconds FLOAT NULL;
```

Comment:
Shouldn't we go with `numeric`?

Reply:
It's been a while since I've used PG so I had to look up the difference.

Seems like `FLOAT` might be more appropriate here since we don't need the absolute precision that `NUMERIC` provides and `FLOAT` is faster/smaller.

Is there some other reason I should consider using `NUMERIC` over `FLOAT` which I might be missing?

Reply:
Passing the database question to @mafredri. My understanding is that floats can always lose precision, so it is safer to store round numbers. If we need fractions of seconds, let's go with milliseconds?

Reply:
I prefer ints as well, but not enough to block this PR, so I'll leave it up to Danny.

Reply:
As discussed in https://github.com/coder/coder/pull/13799#discussion_r1668521931, I actually don't specifically need sub-second precision so I'll be changing over to `integer`.